.github/workflows/docker_publish.yml

```diff
@@ -82,7 +75,7 @@ jobs:
       fail-fast: false
 
     container:
-      image: ghcr.io/${{ github.repository_owner }}/pyne_ubuntu_22.04_py3${{ matrix.hdf5 }}/${{ matrix.stage }}${{ needs.multistage_image_build.outputs.image_tag }}
+      image: ghcr.io/${{ github.repository_owner }}/pyne_ubuntu_22.04_py3${{ matrix.hdf5 }}/${{ matrix.stage }}:latest
```

Comment:
maybe we can use the `output` from the multistage build to set this tag?

Reply:
That's true. Looking at the [tags in the packages list](https://github.com/pyne/pyne/pkgs/container/pyne_ubuntu_22.04_py3%2Fmoab/versions) there does seem to be this `sha256:10b87d094a43f426b103cc7d69842ad57494536973db0824efa111a7049d62d5` tag that would be the output from the multistage action. However, we would still need to export to the `outputs:` of the first job in order to use it in this second job because we are using it in the `container:` section of the second job.